man/chpl.txt

```diff
@@ -195,6 +195,12 @@ OPTIONS
   --[no-]bounds-checks   Enable [disable] run-time bounds checking,
                     e.g. during slicing and array indexing.
 
+  --[no-]formal-domain-checks   Enable [disable] run-time checks to ensure
```

Comment:
The compiler --help says something to that affect, but the man page doesn't. I agree it'd be nice to have the man page say something like that. 

I don't remember Kyle/Ben adding a check that isn't affected by --no-checks. Kyle added the cast-checks, which has a flag. And Ben added additional uses of local-checks. Might be worth double checking with them though